src/domains/rtd-app/04_apim_api_pi.tf

```diff
@@ -0,0 +1,50 @@
+#
+# New Payment Gateway PRODUCTS
+#
+module "payment_instruments_api_product" {
+  source = "git::https://github.com/pagopa/terraform-azurerm-v3.git//api_management_product?ref=v3.0.0"
+
+  count = var.enable.api_payment_instrument ? 1 : 0
+
+  product_id   = "payment_instruments_api_product"
+  display_name = "PAYMENT_INSTRUMENTS_API_PRODUCT"
+  description  = "PAYMENT_INSTRUMENTS_API_PRODUCT"
+
+  api_management_name = data.azurerm_api_management.apim_core.name
+  resource_group_name = data.azurerm_resource_group.apim_rg.name
+
+  published             = true
+  subscription_required = false
+  approval_required     = false
+
+  subscriptions_limit = 0
+
+  policy_xml = templatefile("./api_product/payment_instruments/policy.xml", {})
+
+}
+
+## Payment Instrument registrtion API ##
+module "payment_instruments_interaction" {
+  source = "git::https://github.com/pagopa/terraform-azurerm-v3.git//api_management_api?ref=v3.0.0"
```

Comment:
Could we try to increase version to last available on repo `terraform-azurerm-v3`?